game_eggs/hogwarp/egg-hogwarp.json

```diff
@@ -0,0 +1,112 @@
+            "env_variable": "WINETRICKS_RUN",
+            "default_value": "",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Hogwarp API Key",
+            "description": "A API required to boot - https:\/\/presence.hogwarp.com\/login\r\n- If not set on install it will fail. Set API Key then Reinstall Files",
+            "env_variable": "API_KEY",
+            "default_value": "CHANGEME",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Name",
+            "description": "A name that displays on the Hogwarp list",
+            "env_variable": "SERV_NAME",
+            "default_value": "Ptero Hogwarp Server",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string",
+            "field_type": "text"
+        },
+        {
+            "name": "WineARCH",
+            "description": "Arch type for Wine",
+            "env_variable": "WINEARCH",
+            "default_value": "win64",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "Max players for a server, this depends on your Patreon level for Hogwarp\r\nhttps:\/\/www.patreon.com\/tiltedphoques - ( 4, 8, 16, No Limit ) In that order.",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "4",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|integer",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Icon URL",
+            "description": "The icon that displays on Hogwarps multiplayer listing",
+            "env_variable": "SERV_ICON",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Server Description",
+            "description": "The description that shows on the Multiplayer list",
+            "env_variable": "SERV_DESC",
+            "default_value": "",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "nullable|string",
+            "field_type": "text"
+        },
+        {
+            "name": "Download URL",
+            "description": "URL to pull the files from\r\n- Using GDrive for no bandwidth limits",
+            "env_variable": "DOWNLOAD_URL",
+            "default_value": "http:\/\/dl.dropboxusercontent.com\/scl\/fi\/cd47ho7mhcvt91glm9q6q\/Hogwarp-Server-Files.zip?rlkey=juq8ru988l1yhuc16cwhx2jho&dl=1",
```

Comment:
I am going with a no for this alone. Is this how they distribute the files or is this an file from you?

Reply:
Its just how I was able to get the Download URL to be a direct link - Did you have a sugg for how to run the files?

Theyre distributed through their discord which makes it pulling straight from there a bit of a nightmare